packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3512,6 +3595,24 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
   BuildOwner? get owner => _owner;
   BuildOwner? _owner;
 
+  BuildScope? _buildScope;
+  /// A [BuildScope] within which the [Element] tree should be updated together.
+  ///
+  /// The default implementation returns the parent [Element]'s [buildScope],
```

Comment:
What do you think about renaming _buildScope to _parentBuildScope or _parentBuildScope cache to make it clear that it could be different from the getter `buildScope`.